Model/PayolutionBtwobPaymentMethod.php

```diff
@@ -94,9 +109,59 @@ public function addMandatoryTransactionData(&$oTransaction, $oOrder)
      */
     public function getCheckoutFields()
     {
-        //TODO add company name field
         return [
+            'wdCompanyName' => [
+                'type' => $this->_getCheckoutFieldType(SessionHelper::isCompanyNameSet()),
+                'title' => Helper::translate('wd_company_name_input'),
+                'required' => true,
+            ],
+        ];
+    }
+
+    /**
+     * Checks if the user is older than 18 or the date of birth needs to be entered
+     *
+     * @return bool
+     *
+     * @throws \Exception
+     *
+     * @since 1.3.0
+     */
+    protected function _checkDateOfBirth()
```

Comment:
For me it makes not much sense to check in an B to B payment method for the birth date? do you agree? We need to clear it with @hbrandl

Reply:
We personally agree to this as well, but it was already discussed with Hartwig via phone.

Reply:
We agree to check it if the birthdate is available in the user account (not mandatory to enter a birthdate).